packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -279,6 +301,9 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
     ],
   );
 
+  const handleFormSubmit = (data: VisitFormData, event) =>
+    dayjs(data.visitDate).isBefore(new Date(), 'day') ? handleRetrospectiveVisit(data, event) : onSubmit(data, event);
```

Comment:
Is `day` the right level of granularity for this? What if I'm processing paperwork from a same-day visit? It seems like we knew (at the beginning) whether this was retrospective or not and we should be able to pass that information to this point.